tests/js/sentry-test/loadFixtures.ts

```diff
@@ -145,16 +152,25 @@ export function makeLazyFixtures<UserProvidedFixtures extends Record<any, any>>(
           for (const exportKey in maybeModule) {
             target[exportKey] = maybeModule[exportKey];
           }
-        } catch {
-          // ignore
+        } catch (error) {
+          return () => {
+            throw new Error(
+              `Failed to resolve ${prop} fixture.
+              - Your fixture does not map directly to file on disk or fixture file could be exporting > 1 fixture.
+              - To resolve this, add a mapping to SPECIAL_MAPPING in loadFixtures.ts or ensure fixture export name maps to the file on disk.
+              - If you are seeing this only in CI and you have followed the step above, check the exact casing of the file as it is case sensitive.
+
+              Original Message: ${error}`
```

Comment:
If there's an import error, parse error, or if the file doesn't exist then this'll show it.

The next `throw new Error` will still appear for cases where the named export isn't found in any other case.